packages/api/.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!build
+!package.json
```

Comment:
Fewer files in the build context means faster builds. Also, previously, we were running `npm install` in the Docker image and then also copying node_modules from the host.